components/sections/stats-with-tweets/index.tsx

```diff
@@ -138,11 +138,11 @@ const Tweet = ({ className }: { className?: string }) => {
             height={48}
           />
           <div className="flex flex-col gap-1">
-            <span className="font-semibold leading-none text-neutral-100">
-              User
+            <span className="font-semibold leading-4 text-neutral-100">
+              {username}
             </span>
-            <span className="text-sm font-light leading-none text-neutral-100/60">
-              @handler · Nov 27, 2021
+            <span className="text-sm font-light text-neutral-100/60">
+              {userhandle} · {date}
```

Comment:
We need to use the right data format like `Nov 27, 2021`. You can use [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat), you don't need to modify the json file value but you only need to format it.

Reply:
We also have date-fns' `format` available, wouldn't it be better?

Reply:
```tsx
{userhandle} · {format(date!, "MMM dd, yyy")}
```